pkg/prometheus/stubmetrics_test.go

```diff
@@ -45,28 +45,31 @@ func (s *stubMeterProvider) Meter(name string, opts ...metric.MeterOption) metri
 
 func NewStubMeter(t *testing.T) *stubMeter {
 	return &stubMeter{
-		t:                 t,
-		int64counters:     make(map[string]*stubInt64Counter),
-		float64counters:   make(map[string]*stubFloat64Counter),
-		int64gauges:       make(map[string]*stubInt64ObservableGauge),
-		float64gauges:     make(map[string]*stubFloat64ObservableGauge),
-		int64histograms:   make(map[string]*stubInt64Histogram),
-		float64histograms: make(map[string]*stubFloat64Histogram),
+		t:                       t,
+		int64counters:           make(map[string]*stubInt64Counter),
+		float64counters:         make(map[string]*stubFloat64Counter),
+		int64observablegauges:   make(map[string]*stubInt64ObservableGauge),
+		float64gauges:           make(map[string]*stubFloat64Gauge),
+		float64observablegauges: make(map[string]*stubFloat64ObservableGauge),
```

Comment:
```suggestion
		int64gauges:             make(map[string]*stubInt64Gauge),
		int64observablegauges:   make(map[string]*stubInt64ObservableGauge),
		float64gauges:           make(map[string]*stubFloat64Gauge),
		float64observablegauges: make(map[string]*stubFloat64ObservableGauge),
```

Missing the Int64Gauge to be complete.